Server/JokesOnYou.Web.Api/DTOs/JokeCreateDto.cs

```diff
@@ -7,5 +7,10 @@ namespace JokesOnYou.Web.Api.DTOs
 {
     public class JokeCreateDto
     {
+        public string Premise { get; set; }
+        public string NormalizedPremise { get; set; }
```

Comment:
Whats the purpose of normalizedpremise on the dto?  Shouldnt this just be on the Joke entity and make the premise required then inside its setter you also set the normalizedpremise but value.toupper?